contracts/src/CrossChainApplications/ERC20Bridge/ERC20Bridge.sol

```diff
@@ -464,15 +452,8 @@ contract ERC20Bridge is IERC20Bridge, ITeleporterReceiver, ReentrancyGuard {
         address recipient,
         uint256 amount
     ) private nonReentrant {
-        // Only allow the Teleporter messenger to deliver messages.
-        if (msg.sender != address(teleporterMessenger)) {
```

Comment:
It looks like this check has been removed. Is it no longer necessary?

Reply:
yup check is not needed